yamllint/conf/default.yaml

```
@@ -19,6 +19,11 @@ rules:
     level: warning
   empty-lines: enable
   empty-values: disable
+  float-values:
+    require-numeral-before-decimal: false
+    forbid-scientific-notation: false
+    forbid-nan: false
+    forbid-inf: false
```

Since it's a new rule I prefer it to be completely disabled by default. It enhances performance and reduces risks of bugs.
```suggestion
  float-values: disable
```